guides/user-guide.md

```diff
@@ -0,0 +1,67 @@
+---
+expires: 2019-12-31
+---
+
+# Cyber Security Consulting Team: asking for help
```

Comment:
Style: you appear to have changed the team name from 'consultancy' team to the 'consulting' team.

Reply:
Updated accordingly.